openedx/core/djangoapps/discussions/models.py

```diff
@@ -114,20 +128,30 @@ def pii_sharing_required_message(provider_name):
         'has_full_support': True
     },
     'piazza': {
-        'features': [
-            Features.ANONYMOUS_POSTING.value,
-            Features.BLACKOUT_DISCUSSION_DATES.value,
-            Features.COMMUNITY_TA_SUPPORT.value,
-            Features.DIRECT_MESSAGES_FROM_INSTRUCTORS.value,
-            Features.DISCUSSION_CONTENT_PROMPTS.value,
-            Features.DISCUSSION_PAGE.value,
-            Features.EMAIL_NOTIFICATIONS.value,
-            Features.LTI_BASIC_CONFIGURATION.value,
-            Features.QUESTION_DISCUSSION_SUPPORT.value,
-            Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
-            Features.USER_MENTIONS.value,
-            Features.WCAG_2_0_SUPPORT.value,
-        ],
+        'features': {
+            'basic_support': [
+                Features.DISCUSSION_PAGE.value, # check where this goes
```

Comment:
@awais-ansari not sure which section within features this goes in! Can you verify where to add `DISCUSSION_PAGE`!

Reply:
@saadyousafarbi I do not know. I think @marcotuts can help you with this.